runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -133,36 +138,41 @@ public void updateConfig(ConfigChanged event)
 	@Subscribe
 	public void onChatMessage(ChatMessage event)
 	{
-		if (event.getType() != ChatMessageType.SERVER || !config.showBindNeck())
+		if (event.getType() != ChatMessageType.SERVER && config.showBindNeck())
```

Comment:
Not sure why this was changed in this way. At the very least, a guard clause for non-server messages should still remain:

```
if (event.getType() != ChatMessageType.SERVER)
{
    return;
}
```